hushline/model.py

```diff
@@ -349,3 +411,96 @@ def __init__(self) -> None:
 
     def __repr__(self) -> str:
         return f"<InviteCode {self.code}>"
+
+
+# Paid tiers
+class Tier(Model):
+    __tablename__ = "tiers"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    name: Mapped[str] = mapped_column(db.String(255), unique=True)
+    monthly_amount: Mapped[int] = mapped_column(db.Integer)  # in cents USD
+    stripe_product_id: Mapped[Optional[str]] = mapped_column(db.String(255), unique=True)
+    stripe_price_id: Mapped[Optional[str]] = mapped_column(db.String(255), unique=True)
+
+    def __init__(self, name: str, monthly_amount: int) -> None:
+        super().__init__()
+        self.name = name
```

Comment:
Oneliner -> `super().__init__(name=name, monthly_amount=monthly_amount)`. Though `mypy` might yell about that. :thinking:

Reply:
mypy yells at me :(